WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/viewholders/RestoreAdditionalInformationViewHolder.kt

```diff
@@ -0,0 +1,19 @@
+package org.wordpress.android.ui.jetpack.restore.viewholders
+
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.backup_list_additional_information_item.*
+import org.wordpress.android.R
+import org.wordpress.android.ui.jetpack.common.JetpackListItemState
+import org.wordpress.android.ui.jetpack.common.viewholders.JetpackViewHolder
+import org.wordpress.android.ui.jetpack.restore.RestoreListItemState.AdditionalInformationState
+import org.wordpress.android.ui.utils.UiHelpers
+
+class RestoreAdditionalInformationViewHolder(
+    private val uiHelpers: UiHelpers,
+    parent: ViewGroup
+) : JetpackViewHolder(R.layout.backup_list_additional_information_item, parent) {
```

Comment:
Warning (⚠️): This `Restore` related view holder is using a `Backup` related layout file. This is most probably an oversight, right?

PS: The same applies for the `RestoreProgressViewHolder` class.

Reply:
That is correct - "backup" is generic - I will be renaming to backup_restore later today :)